conda_build/build.py

```diff
@@ -3235,7 +3235,7 @@ def _write_test_run_script(
             tf.write(
                 '"{perl}" "{test_file}"\n'.format(
                     perl=metadata.config.perl_bin(
-                        metadata.config.test_prefix, metadata.config.host_platform
+                        metadata.config.test_prefix, sys.platform
```

Comment:
Shouldn't this be `metadata.config.build_platform`?

Reply:
Not entirely sure, but I can't see where that is defined or used in the codebase. `sys.platform` seems to be used in a number of places.